GitTfs/Commands/Clone.cs

```diff
@@ -160,6 +162,14 @@ public int Run(string tfsUrl, string tfsRepositoryPath, string gitRepositoryPath
             return retVal;
         }
 
+        private string HideUserCredeentials(string commandLineRun)
+        {
+            Regex rgx = new Regex("(--username|-u)=[^ ]+");
```

Comment:
I think the `=` is optional. For example `git tfs clone --username spraints --password SECRETOMG https://topsecret.com/tfs $/reallysupersecret`. It'd be super super duper to have a test for just this function, with a variety of command line forms to demonstrate that it's doing what it's supposed to.